src/it/scala/com/wavesplatform/it/transactions/TransferTransactionSpecification.scala

```diff
@@ -52,6 +60,79 @@ class TransferTransactionSpecification(override val allNodes: Seq[Node]) extends
     Await.result(f, 1 minute)
   }
 
+  test("invalid signed waves transfer should not be in UTX or blockchain") {
```

Comment:
Still no need for this code,  just create a transaction, then extract everything you need to `txId` and `invalidByTsSignedRequest`, this will reduce up the code up to 50 lines